lib/MusicBrainz/Server/Controller/CDTOC.pm

```diff
@@ -253,10 +254,23 @@ sub attach : Local DenyWhenReadonly
             $c->detach;
         }
         elsif ($search_release->submitted_and_valid($c->req->query_params)) {
+            my $query = $search_release->field('query')->value;
             my $releases = $self->_load_paged($c, sub {
-                $c->model('Search')->search('release', $search_release->field('query')->value, shift, shift,
+                if (is_guid($query)) {
```

Comment:
Ideally this would check if it contains a GUID like most of our other searches, so you can still paste a release link directly, but this is certainly an improvement.

Reply:
Changed it to use a regular expression to extract the MBID, which'll allow people to paste a link.